docs/modules/ROOT/pages/usage.adoc

```diff
@@ -0,0 +1,40 @@
+= Usage
+
+The cluster is configured via a YAML file.
+
+[source,yaml]
+----
+apiVersion: nifi.stackable.tech/v1
+kind: NiFiCluster
+metadata:
+  name: simple-nificluster
+spec:
+version: "1.13.2"
+zookeeperReference:
+  name: simple
+  namespace: default
+  chroot: /nifi
+nodes:
+  selectors:
+    default:
+      selector:
+        matchLabels:
+          kubernetes.io/hostname: nifi-node-1
+        instances: 1
+        instancesPerNode: 1
+      config:
+        httpPort: 10000
+        nodeProtocolPort: 10443
+        nodeLoadBalancingPort: 6342
+    default2:
+      selector:
+        matchLabels:
+          kubernetes.io/hostname: nifi-node-2
+        instances: 1
+        instancesPerNode: 1
```

Comment:
This was already wrong, but "instances" and "instancesPerNode" should be on the same level as "selector"